amdih/cp2k/container.yaml

```diff
@@ -0,0 +1,35 @@
+docker: amdih/cp2k
+url: https://www.amd.com/en/technologies/infinity-hub/cp2k
+description: CP2K is a quantum chemistry and solid state physics software package that can perform atomistic simulations of solid state, liquid, molecular, periodic, material, crystal, and biological systems. 
+maintainer: "@cristiandipietrantonio"
+latest:
+  '8.2': sha256:5947603de32c4e690f734075f7625563a2de18d1c3276ec10fceccdb332022ec
+tags:
+  '8.2': sha256:5947603de32c4e690f734075f7625563a2de18d1c3276ec10fceccdb332022ec
+aliases:
+- name: cp2k.popt
```

Comment:
You just fixed the path as I was writing a comment - very speedy!

Another question - is there any reason you aren't putting full paths? We typically put full paths to the executables (aliases).

Reply:
Hi @vsoch, no particular reason. I will update the recipes when I have a bit of time, thanks for the suggestion.